### DISCUSSION TITLE
Vlad Logo.

### DISCUSSION THREAD

**Discussion by FullBleed** [vladmandic/sdnext#879] [Ideas]
I think enough work is going into this fork that it could use its own logo.

So I threw together something that would look "related" to the A1111 logos... didn't get too crazy with it.

Probably not the best idea to keep naming these things after the repo owners, but until this get rebranded...

![vlad-512](https://github.com/vladmandic/automatic/assets/37131742/a8b02f15-487b-47f9-953e-9ee8e2f44261)

![vlad fade](https://github.com/vladmandic/automatic/assets/37131742/a7c7db3b-ce30-4a95-8671-5c54890c57c5)

**Comment by vladmandic** [comment]
I love the initiative!
But honestly, I hate my name to be used as app name - I should have came up with actual name from day0.
I'm playing with idea SD.Next (as soon it will support other models as well, not just regular SD).
If you play with that idea, I'm curious what you'd come up with (I actually like your design, just don't want to have my name as app name)

**Reply by Averytingwong** [reply]
I'm a big fan of SD Next.

**Reply by FullBleed** [reply]
> I love the initiative! But honestly, I hate my name to be used as app name - I should have came up with actual name from day0. I'm playing with idea SD.Next (as soon it will support other models as well, not just regular SD). If you play with that idea, I'm curious what you'd come up with (I actually like your design, just don't want to have my name as app name)

Understood, and I appreciate the humility.

I was kicking around some ideas, but I kept thinking that it really should be something "dynamic".  And what's more dynamic than SD itself?

So my thoughts kept drifting to the "ControlNet S Challenge": https://www.reddit.com/r/StableDiffusion/comments/1183gx1/controlnet_cool_s_challenge_use_scribble_mode_to/

Why not just pick a basic "Next icon" from an image search (any right facing arrow/triangle icon) for the core logo.... and solicit a ControlNet challenge on it?

Then you could have a default logo, ones specific to themes, randomized, user selected, etc.

**Reply by vladmandic** [reply]
cool idea - but i'm not sure if triggering a job on startup is the way to go, there are plenty of users running on cpu-only and it would take significant time :)
but i'd be more than ok with pool of pre-created ones to show randomly and one as icon.

**Reply by ghost** [reply]
![SDNEXT-01](https://github.com/vladmandic/automatic/assets/66656514/0ae83239-283d-480d-9eb9-b39d31252ab1)
Accidentally replied to the wrong person oops, but I took a swing at a logo design.

**Comment by drax-xard** [comment]
Automandic Diffusion lol

**Comment by brknsoul** [comment]
Prompt: `A business logo for "SD.Next"`

![image](https://github.com/vladmandic/automatic/assets/1969381/72db1c76-d716-4f6f-a9aa-8bae31245816)


Prompt: `A business logo for "SD.Next" using red and purple as primary colours, black background.`

![image](https://github.com/vladmandic/automatic/assets/1969381/c5a4070a-1b5f-4606-b9ef-deefc1d6a028)


I love how SD tries _really_ hard to do text. XD

**Comment by mrt0dd** [comment]
love it
